### ISSUE TITLE
"Error: Failed to get key index" when the regex contains an escaping backslash

### ISSUE THREAD

**Issue by ogizanagi** [kanga333/variable-mapper#16]
In case of such an expression:

```json
"^staging(-[\\w/-]*)?$": {
   ...
}
```

We get:

```shell
Error: Failed to get key index of ^staging(-[\w/-]*)?$
```

Which I guess is related to the fact [the raw JSON is used to find the index](https://github.com/kanga333/variable-mapper/blob/8a099c99e52060d602dd343d8fa60aed5120edc0/src/mapper.ts#L129-L132) for ordering purposes.

**Comment by kanga333** [comment]
Thanks for reporting this!
This is a bug in the process to guarantee the order of keys in JSON. I will fix this when I have time.

**Comment by kanga333** [comment]
@ogizanagi  Thank you for your patience.
I've fixed the issue and released it as [v0.2.2](https://github.com/kanga333/variable-mapper/releases/tag/v0.2.2).
